code/lib/types/src/modules/api-stories.ts

```diff
@@ -184,3 +184,7 @@ export interface API_StatusObject {
 
 export type API_StatusState = Record<StoryId, Record<string, API_StatusObject>>;
 export type API_StatusUpdate = Record<StoryId, API_StatusObject>;
+
+export type API_FilterFunction = (
+  item: API_IndexHash[keyof API_IndexHash] & { status: Record<string, API_StatusObject> }
```

Comment:
It seems like some properties are missing from the Item type, e.g. tags. Is that intentional? Maybe we can add them? Using tags for filtering seems quite useful

Reply:
I'm guessing that would have to be added here:
https://github.com/storybookjs/storybook/blob/cda5eed4d8cbf96957a3a5c0518ab70547895a56/code/lib/types/src/modules/api-stories.ts#L82-L102

But I think the reason it's not there, is because we aren't committed to tags being a public API?
I'm not sure, but it seems a discussion best had not in this PR

Reply:
Makes sense to me!

Reply:
>I'm not sure, but it seems a discussion best had not in this PR

Speaking of which, has there been an RFC process for the changes here?  I ask because it does seem to be closely related to what tags were supposed to accomplish, and it would be a bummer if we release an API which has to be significantly changed or removed if/when tags are rolled out.